### ISSUE TITLE
Integration tests fail on CreateContainerWithGroupAddTest

### ISSUE THREAD

**Issue by paulbellamy** [docker/docker-py#769]
```sh
$ docker run -e NOT_ON_HOST=true -e DOCKER_HOST=tcp://172.17.42.1:2375 -v /tmp:/tmp -v /var/run/docker.sock:/var/run/docker.sock joffrey/docker-py py.test tests/integration_test.py::CreateContainerWithGroupAddTest
============================= test session starts ==============================
platform linux2 -- Python 2.7.10 -- py-1.4.30 -- pytest-2.7.2
rootdir: /home/docker-py, inifile:
plugins: cov
collected 2 items

tests/integration_test.py FF

=================================== FAILURES ===================================
______________ CreateContainerWithGroupAddTest.test_group_id_ints ______________

self = <tests.integration_test.CreateContainerWithGroupAddTest testMethod=test_group_id_ints>

    def test_group_id_ints(self):
        container = self.client.create_container(
            'busybox', 'id -G',
            host_config=self.client.create_host_config(group_add=[1000, 1001])
        )
        self.tmp_containers.append(container)
        self.client.start(container)
        self.client.wait(container)

        logs = self.client.logs(container)
        if six.PY3:
            logs = logs.decode('utf-8')
        groups = logs.strip().split(' ')
>       self.assertIn('1000', groups)
E       AssertionError: '1000' not found in ['0', '10']

tests/integration_test.py:290: AssertionError
____________ CreateContainerWithGroupAddTest.test_group_id_strings _____________

self = <tests.integration_test.CreateContainerWithGroupAddTest testMethod=test_group_id_strings>

    def test_group_id_strings(self):
        container = self.client.create_container(
            'busybox', 'id -G', host_config=self.client.create_host_config(
                group_add=['1000', '1001']
            )
        )
        self.tmp_containers.append(container)
        self.client.start(container)
        self.client.wait(container)

        logs = self.client.logs(container)
        if six.PY3:
            logs = logs.decode('utf-8')
        groups = logs.strip().split(' ')
>       self.assertIn('1000', groups)
E       AssertionError: '1000' not found in ['0', '10']

tests/integration_test.py:307: AssertionError
=========================== 2 failed in 8.47 seconds ===========================
```

**Comment by rade** [comment]
I believe the test was run against Docker 1.7.1. Could that be the issue, i.e. is the new test depending on some Docker 1.8+ functionality?

**Comment by shin-** [comment]
Oh, yeah. It's only available in 1.8+. I'll add a skip. Thanks for the report.

**Comment by shin-** [comment]
Fixed via #764
